create_sql_jet.go

```diff
@@ -0,0 +1,75 @@
+package generatorjetmysql
```

Comment:
Package Main is in the folder ./src/. I would have to move the file in that folder. Since its not techically part of the app, just the setup. The create_sql_jet.sh file is also not in the ./src/ folder